src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -1314,7 +1314,7 @@ public virtual void HandlePackageUninstall(PackageResult packageResult, Chocolat
         {
             if (!_fileSystem.DirectoryExists(packageResult.InstallLocation))
             {
-                packageResult.InstallLocation += ".{0}".FormatWith(packageResult.PackageMetadata.Version.ToStringSafe());
+                packageResult.InstallLocation += ".{0}".FormatWith(packageResult.PackageMetadata.Version.ToNormalizedStringChecked());
             }
 

```

Comment:
I think we can remove this entire if statement.

This looks to be related to side by side installations, which we no longer support.

/cc @gep13

Reply:
Yes, I would tend to agree.  There should be no need to have a versioned folder name going forwad.

Reply:
The question would be if we should remove it right now or not.

In theory (need to verify the behavior first), leaving it in could allow users to uninstall a package that was previously in v1 installed in side by side.

Removing it, will disable that ability immediately.

If we keep it, we should not call `ToNormalizedStringChecked` though.

Reply:
I have removed this statement for now, as it doesn't have any actual effect.